test/test/jfr/Cache.java

```diff
@@ -35,7 +33,7 @@ public String get(Long key) {
         return vt.value;
     }
 
-    private void calculateTop() {
+    protected void calculateTop() {
```

Comment:
If the intent is to make it package-visible only, I'd rather use the default visibility:
```suggestion
    void calculateTop() {
```
`protected` kinda tells "I'm going to extend this class"